LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Modified BSD License
```

Comment:
We don't seem to satisfy any of the conditions listed there for instant approval.

And even if we did... we presumably need approval from the NIH before officially setting a license.

If we have to say something, I'd recommend something that says "We're still figuring out our licensing... come back later." Not sure what the right way of doing that is.